mobx/lib/src/core/reaction.dart

```diff
@@ -64,7 +64,7 @@ class ReactionImpl implements Reaction {
     _isRunning = false;
 
     if (_isDisposed) {
-      _context._clearObservables(this);
+      _context.clearObservables(this);
```

Comment:
can we cover this use case with a test ?

Reply:
@pavanpodila done